lib/phase/flatten.js

```diff
@@ -3,6 +3,9 @@
 const Hoek = require('@hapi/hoek');
 const clone = require('clone');
 
+const STAGE_PREFIX = 'stage@';
+const DEFAULT_JOB = { image: 'node:18', steps: [{ noop: 'echo noop' }] };
```

Comment:
Instead of hardcoding the image to `node:18`, can we use tag? something like `node:current` / `node:latest`
```suggestion
const DEFAULT_JOB = { image: 'node:18', steps: [{ noop: 'echo noop' }] };
```

Reply:
I think it's better to hardcode so we are sure we support that version.